src/logics/LALogic.cc

```diff
@@ -184,6 +189,30 @@ lbool LALogic::retrieveSubstitutions(const vec<PtAsgn>& facts, Map<PTRef,PtAsgn,
     }
     return arithmeticElimination(top_level_arith, substs);
 }
+
+uint32_t LessThan_deepPTRef::getMin(PTRef tr) const {
+    PTRef c_t;
+    PTRef v_t;
+    l.splitTermToVarAndConst(tr, v_t, c_t);
+    return v_t.x;
+}
+
+bool LessThan_deepPTRef::operator()(PTRef& x_, PTRef& y_) {
```

Comment:
I think this should be either `const` references, or maybe even better directly `PTRef`s, as copies, since it is basically just a 32-bit number, which should be better that passing a reference, which is (typically) 64-bit pointer.